config/crd/patches/webhook_in_mysqlusers.yaml

```diff
@@ -5,6 +5,7 @@ kind: CustomResourceDefinition
 metadata:
   name: mysqlusers.azure.microsoft.com
 spec:
+  preserveUnknownFields: false
```

Comment:
Is there a story here? Wondering why we need to be explicit here but I don't see the same explicitness in other webhook defs (for example azuresqlusers)

Reply:
Hmm - I copied it from the other webhook patches (eg MySQLServer) after getting an error on deploy. The only ones that have it are the enabled ones (AzureSQLUser only has one version) - I'm going to add it to all of them to save the next person (or future me) the same confusion.

Reply:
Ha, although I can understand your confusion, since AzureSQLUser *does* have webhooks, just not conversions.